phusion/agent/Dockerfile

```diff
@@ -7,9 +7,10 @@ ARG GO_VERSION="16.8.0-3929"
 ARG DOWNLOAD_URL="https://download.gocd.io/binaries"
 
 RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
+RUN add-apt-repository ppa:openjdk-r/ppa
```

Comment:
should be `add-apt-repository -y ppa:openjdk-r/ppa` else it would ask for a prompt
Don't need `software-properties-common python-software-properties` to be installed. software-properties-common is required to run `add-apt-repository`, but that seems to be available already
Make it `openjdk-8-jre-headless`